src/components/i18n.js

```diff
@@ -1,65 +0,0 @@
-import i18n from 'i18next';
```

Comment:
I made changes in this file that are available in dev. No major changes, but having those changes in your new file would be great.

Reply:
I think they're all incorporated

Reply:
Even the comment?

Reply:
Yeah I copied and pasted the comment over when resolving the merge conflicts